subworkflows/local/prepare_references.nf

```diff
@@ -43,16 +43,16 @@ workflow PREPARE_REFERENCES {
         BUILD_DICT(ch_fasta)
         ch_dict = BUILD_DICT.out.dict.collect()
 
-        gtf_meta = Channel.fromPath(gtf).map{ it -> [ [id:it[0]], it ] }.collect()
+        gtf_meta = Channel.fromPath(gtf).map{ it -> [ [id:it[0]], it ] }
```

Comment:
```suggestion
        ch_gtf = Channel.fromPath(gtf).map{ it -> [ [id:it[0]], it ] }.collect()
```
could we call this `ch_gtf` directly?

Reply:
On a related note, is there a reason we have added meta to gtf channel? It seems like all the modules used in the pipeline used to take gtf file without a meta?

Reply:
As the standards are to give annotation files with meta, I have added it to local modules and tried to always have all references with it to try and keep it simple (or less complicated). I think its only Salmon that does not have meta now

Reply:
I just realised I didn't answer your question @jemten, no, we can't because if the file is .gz it will have to go through GUNZIP

Reply:
There might be a way around it but let's save it for later 😊